src/fsevents/fsevents_stubs.c

```diff
@@ -261,20 +261,22 @@ static const FSEventStreamEventFlags action_mask =
     kFSEventStreamEventFlagItemCreated | kFSEventStreamEventFlagItemRemoved |
     kFSEventStreamEventFlagItemRenamed | kFSEventStreamEventFlagItemModified;
 
+static int count_set_bits(uint32_t bits) {
+  // XXX slow
```

Comment:
I suppose we don't need to be very fast, but `__builtin_popcount(bits)` should do the tricks for the compilers we support on mac.

Reply:
Good call. We can try that out in a separate PR.

Reply:
did so in #6224